### PR TITLE
Bugfix/misc changes additions

### PR DESCRIPTION
**Pull request by KhalidAdan**
Miscellaneous changes / additions:

- Added a hyperlink to the privacy notice and added the correct translation
- removed instances of FAQ in the app
- Added table translations
- Removed the income field debounce to try and assuage the bug with the form resetting itself, but I cannot repro the issue reliable
- Fixed the back button to have the correct text (was next, incorrectly)
- Bold the age ranges on the home page
- Updated the GIS text on the home page
- Updated the tooltips max-width per Alla's request
- updated the border colour for the tab to be consistent with all the other border and content colouring changes from last week
- Next steps links are now an array and will render the correct amount in mobile (stacked) and desktop (side by side)

changes shoved in by Jeremy:
- add "apply" link for GIS
- title case app name in french into page
- update french livingCountry tooltip
- make partner income tooltip equal to client income tooltip
- update link text for french oasRetroactive

### CODE REVIEW

File: client-state/models/Form.ts [hunk 1]
```diff
@@ -187,6 +187,7 @@ export const Form = types
           field.setError(d.message)
         }
       } else {
+        console.log(data)
```
Comment (JeremyKennedy) [inline client-state/models/Form.ts:190]: 🤔

**Review by JeremyKennedy — APPROVED** [review]
Looks great! I love easy-to-review PRs. All the diffs are straightforward, lots of quick wins here!

**Review by JeremyKennedy — CHANGES_REQUESTED** [review]
I take that back!!!

image

**Comment by KhalidAdan** [comment]
Fixed! Think we could get this out today before EE testing? @JeremyKennedy

**Review by JeremyKennedy — APPROVED** [review]

**Pull request merged**